common/protob/messages-management.proto

```diff
@@ -102,6 +102,7 @@ message Features {
     optional SafetyCheckLevel safety_checks = 37;            // safety check level, set to Prompt to limit path namespace enforcement
     optional uint32 auto_lock_delay_ms = 38;    // number of milliseconds after which the device locks itself
     optional uint32 display_rotation = 39;      // in degrees from North
+    optional bool experimental_features = 40;   // are experimental message types enabled?
```

Comment:
Want me to change it to `developer_mode` or is that name only for displaying in the confirmation dialog?

Reply:
I think the name here should match whatever it will be called in the dialog. However let's first see what @matejcik and @tsusanka have to say about "developer mode" vs. "experimental features". I don't actually have a strong preference for either.